components/molecule/autosuggest/src/index.js

```diff
@@ -53,19 +55,24 @@ const MoleculeAutosuggest = ({
   onSelect = () => {},
   onToggle = () => {},
   refMoleculeAutosuggest: refMoleculeAutosuggestFromProps,
-  refMoleculeAutosuggestInput: refMoleculeAutosuggestInputFromProps = {},
+  refMoleculeAutosuggestInput: refMoleculeAutosuggestInputFromProps,
   state,
   ...restProps
 }) => {
-  const refMoleculeAutosuggest = useRef(
-    refMoleculeAutosuggestFromProps?.current
+  const innerRefMoleculeAutosuggest = useRef(null)
```

Comment:
Why set as null?

The other prop innerRefMoleculeAutosuggestInput is set as useRef()

Reply:
done